### ISSUE TITLE
Can I test both hands?

### ISSUE THREAD

**Issue by MLsmaller** [xinghaochen/Pose-REN#17]
I want to detect the two hands of a picture at the same time. Firstly, I will find out the bounding boxes of the two hands with the method of object detection, and then I will bring out the depth information of the picture in the two enveloping boxes and use the model to detect them respectively. Is this feasible?

**Comment by xinghaochen** [comment]
Yes it is feasible. You should detect two hands, crop them properly and use the model to estimate hand pose for each hand. Note that the models are trained using the right hand, you have to flip the left hand horizontally to estimate the hand pose.
